project_task_name_with_id/models/project_task.py

```diff
@@ -9,9 +9,8 @@ class ProjectTask(models.Model):
 
     @api.model
     def name_search(self, name, args=None, operator="ilike", limit=100):
-        recs = self.search(
-            ["|", ("name", operator, name), ("id", operator, name)], limit=limit
-        )
+        args = (args or []) + ["|", ("name", operator, name), ("id", operator, name)]
+        recs = self.search(args, limit=limit)
```

Comment:
Thanks for the fix! This may be safer, perhaps?
```suggestion
        domain = expression.AND(
            [args or [], ["|", ("name", operator, name), ("id", operator, name)]]
        )
        recs = self.search(domain, limit=limit)
```

Reply:
@yostashiro done! Thanks!